### ISSUE TITLE
Parsing error: '>' expected in .html files

### ISSUE THREAD

**Issue by developer239** [angular-eslint/angular-eslint#81]
When I run eslint using your builder I get following error in HTML files:

`Parsing error: '>' expected`

I don't like example configuration:

https://github.com/angular-eslint/angular-eslint/blob/master/packages/integration-tests/fixtures/angular-cli-workspace/.eslintrc.js

Why do you use overrides for everything?

Is it somehow possible to extend this configuration and use it in my project?

I have
- [base eslint configuration](https://github.com/code-quality-resources/eslint-config-base)

Then I have 
- [typescript eslint configuration](https://github.com/code-quality-resources/eslint-config-typescript) which extends base configuration

Now I created
- [angular configuration ](https://github.com/code-quality-resources/eslint-config-angular) with your amazing plugin and I wanted to extend my typescript eslint configuration. However, I can't make it work with HTML files.

What am I missing? 🙂 🙏 To me, extensibility is important because I want to share basic eslint and typescript rules across all node.js, react, react-native and angular projects.

My current solution is replacing lint command with `eslint --ext .ts src` but this way I don't lint HTML files at all.

**Comment by JamesHenry** [comment]
We have shareable configs now as of `v0.0.1-alpha.32`

Take a look at: `packages/integration-tests/fixtures/angular-cli-workspace/.eslintrc.js` and `packages/eslint-plugin/src/configs/README.md`

And please check out https://github.com/angular-eslint/angular-eslint/issues/79 regarding usage of this tooling without the Angular CLI

**Comment by developer239** [comment]
@JamesHenry Thank you for your time 🙌  but you probably didn't read my issue at all because I linked the same documentation that you did. 🙂 

I tried different configurations and the problem is probably in my misunderstanding how eslint works.

I will use your HTML config in separate plugin and I will run that plugin only on HTML files that way I can extend my existing eslint configuration + use your plugin + lint HTML files.
